src/main/java/org/dependencytrack/notification/NotificationRouter.java

```diff
@@ -89,11 +89,11 @@ private List<NotificationRule> resolveRules(final Notification notification) {
 
             final NotificationLevel level = notification.getLevel();
             if (NotificationLevel.INFORMATIONAL == level) {
-                sb.append("(notificationLevel == 'INFORMATIONAL' || notificationLevel == 'WARNING' || notificationLevel == 'ERROR') && ");
+                sb.append("notificationLevel == 'INFORMATIONAL' && ");
             } else if (NotificationLevel.WARNING == level) {
-                sb.append("(notificationLevel == 'WARNING' || notificationLevel == 'ERROR') && ");
+                sb.append("(notificationLevel == 'WARNING' || notificationLevel == 'INFORMATIONAL') && ");
```

Comment:
* If a notification with level `INFORMATIONAL` comes in, we want all rules with level `INFORMATIONAL` or lower. No lower level exists, so filtering for `INFORMATIONAL` only is fine.
* If a notification with level `WARNING` comes in, we want all rules with level `WARNING` or lower. `INFORMATIONAL` < `WARNING`, so that is fine as well.
* If a notification with level `ERROR` comes in, we want all rules with level `ERROR` or lower. `INFORMATIONAL` < `WARNING` < `ERROR`. 

What am I missing?

Reply:
If you read my Pull Request comment, that was correct, the code second line is wrong. 

> If a notification with level WARNING comes in, we want all rules with level WARNING or lower. INFORMATIONAL < WARNING, so that is fine as well.

WARN should only show WARN and ERROR, not WARN and INFO.